runtime.txt

```diff
@@ -0,0 +1 @@
+python-3.8.11
```

Comment:
@iTaylorCodes what is the use of this setup ?

Reply:
This runtime.txt file was to tell Heroku which version of python to use. Removed file.